doc/api/process.md

```diff
@@ -518,11 +518,15 @@ process.on('SIGTERM', handle);
    the process hanging in an endless loop, since listeners attached using
    `process.on()` are called asynchronously and therefore unable to correct the
    underlying problem.
+* `0` Sending signal `0` can be used to test for the existence of a process, it
+   will error if the process does not exist.
 
 Windows does not support sending signals, but Node.js offers some emulation
-with [`process.kill()`][], and [`subprocess.kill()`][]. Sending signal `0` can
-be used to test for the existence of a process. Sending `SIGINT`, `SIGTERM`,
-and `SIGKILL` cause the unconditional termination of the target process.
+with [`process.kill()`][], and [`subprocess.kill()`][]:
+* Sending `SIGINT`, `SIGTERM`, and `SIGKILL` cause the unconditional termination
+  of the target process, and subprocess will report that the process was
+  terminated with the respective signal (even though it did not).
```

Comment:
The `(even though it did not)` part is confusing, it sounds like the process will not get terminated.